internal/versions/versions.go

```diff
@@ -266,6 +269,16 @@ func (h *handler) validateVersions() error {
 		if osImage.Version == nil {
 			return errors.Errorf(fmt.Sprintf(missingValueTemplate, "Version"))
 		}
+
+		missingValueTemplate = "Missing value in ReleaseImage for '%s' field"
```

Comment:
The check below implies that we will have a default architecture. IIRC, the manifests list API doesn't allow for setting a default when using multi-arch repositories, which means we would have to pick the default.

Can we document that `x86_64` will be the default architecture? 

TBH, I would be inclined to make the `cpu_architecture` field required. It is better to be explicit and it will simplify the implementation, debugging, and maintenance.

Reply:
Sure, makes sense. I'll add a validation for the architecture as well, can be mandatory when using this list.